modules/oadp-1-5-0-release-notes.adoc

```diff
@@ -0,0 +1,223 @@
+
+* `cacheLimitMB`: It sets the local data cache size limit in megabytes.
+* `fullMaintenanceInterval`: The default value is 24 hours. Controls the removal rate of deleted Velero backups from the Kopia repository using the following override options:
+** `normalGC: 24 hours`
+** `fastGC: 12 hours`
+** `eagerGC: 6 hours`
+
+link:https://issues.redhat.com/browse/OADP-5900[OADP-5900]
+
+.Enhancing the node-agent security
+
+With this update, the following changes are added:
+
+* A new `configuration` option is now added to the `velero` field in the DPA specification.
+* The default value for the `disableFsBackup` parameter is `false` or `non-existing`, which is the same as false. With this, the following options are added to the `SecurityContext`:
+** `Privileged: true`
+** `AllowPrivilegeEscalation: true`
+* With the `disableFsBackup`parameter set to `true`, removes the following mounts from the node-agent:
+** `host-pods`
+** `host-plugins`
+* Modifies that the node-agent is always run as a non-root user.
+* Changes the root file system to read only.
+* Updates the following mount points with the write access:
+** `/home/velero`
+** `tmp/credentials`
+* Uses the `SeccompProfileTypeRuntimeDefault` option for the `SeccompProfile` parameter.
+
+link:https://issues.redhat.com/browse/OADP-5031[OADP-5031]
+
+.Adds DPA support for parallel item backup
+
+By default, only one thread processes an item block. Velero 1.16 supports a parallel item backup, where multiple items within a backup can be processed in parallel. 
+You can use the optional Velero server parameter `--item-block-worker-count`to run additional worker threads to process items in parallel. To enable this in OADP, set the `dpa.Spec.Configuration.Velero.ItemBlockWorkerCount` parameter to an integer value greater than zero.
+
+link:https://issues.redhat.com/browse/OADP-5635[OADP-5635]
+
+.OADP logs are now available in the JSON format
+
+With the of release {oadp-short} 1.5.0, the logs are now available in the JSON format. It helps to have pre-parsed data in their Elastic logs management system.
+
+link:https://issues.redhat.com/browse/OADP-3391[OADP-3391]
+
+.The `oc get dpa` command now displays Status
+
+With this release, the `oc get dpa` command now displays Status to improve user experience.
+
+link:https://issues.redhat.com/browse/OADP-1338[OADP-1338]
+
+[id="resolved-issues-1-5-0_{context}"]
+== Resolved issues
+
+.Containers now use `FallbackToLogsOnError` for `terminationMessagePolicy`
+
+With this release, for all the {oadp-first} Operator containers such as `operator-manager`, `velero`, `node-agent`, and `non-admin-controller`, the `terminationMessagePolicy` field can now set to the `FallbackToLogsOnError` value. This change ensures that if a container exits with an error and the termination message file is empty, {OCP-short} will use the last portion of the container's log output as the termination message.
```

Comment:
```suggestion
With this release, the `terminationMessagePolicy` field can now set the `FallbackToLogsOnError` value for the {oadp-first} Operator containers such as `operator-manager`, `velero`, `node-agent`, and `non-admin-controller`. This change ensures that if a container exits with an error and the termination message file is empty, {OCP-short} uses the last portion of the container logs output as the termination message.
```

do you mean this? ----
"With this release, the `terminationMessagePolicy` field can now set the `FallbackToLogsOnError` value for the {oadp-first} Operator containers such as `operator-manager`, `velero`, `node-agent`, and `non-admin-controller`. "